README.md

```diff
@@ -128,6 +126,18 @@ To save DataFrame as avro you should use the `save` method in `AvroSaver`. For e
 ```scala
 scala> AvroSaver.save(myRDD, "my/output/dir")
 ```
+
+To include aliases column in scheme invoke the method `addAvroAliasColumns()` of DataFrame.
+With alias while saving `saveAsAvroFile`, alias columns in DataFrame schema will not be include in Avro file.
+Alias will be available in `aliases` of avro schema.
+```scala
+scala>val dfWithAlias = df.addAvroAliasColumns()
+```
+In Spark SQL, loading avro with alias columns use:
+```
+create table mytable using com.databricks.spark.avro OPTIONS(path "test.avro", withAlias "true");
+```
+
```

Comment:
@marmbrus user can set alias and docs to a avro using avro tools. Not using spark-avro